packages/orbit-components/src/Drawer/index.tsx

```diff
@@ -200,41 +70,74 @@ const Drawer = ({
       }
     }
 
-    const handleKeyDown = ev => {
-      if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
-        onClose();
-      }
-    };
-
     window.addEventListener("keydown", handleKeyDown);
     return () => window.removeEventListener("keydown", handleKeyDown);
-  }, [overlayShown, setOverlayShown, setOverlayShownWithTimeout, shown, onClose]);
+  }, [overlayShown, setOverlayShown, setOverlayShownWithTimeout, handleKeyDown, shown, onClose]);
+
+  const vars = {
+    "--lm-drawer-width": width,
+  };
+
+  const varClasses = [vars["--lm-drawer-width"] != null && "lm:max-w-[var(--lm-drawer-width)]"];
+
+  const onlyIcon = !title && !actions;
+  const bordered = !!(title || actions);
 
   return (
-    <StyledDrawer
+    <div
       role="button"
-      overlayShown={overlayShown}
-      shown={shown}
+      tabIndex={0}
+      onKeyDown={handleKeyDown}
+      className={cx(
+        "orbit-drawer",
+        "flex",
+        "fixed bottom-0 left-0 right-0 top-0",
+        "h-full w-full",
+        "z-drawer",
+        "duration-fast transition-colors ease-in-out",
+        overlayShown ? "visible" : "invisible",
+        shown ? "bg-drawer-overlay-background" : "transparent",
+      )}
       onClick={handleOnClose}
       data-test={dataTest}
       id={id}
       aria-hidden={!shown}
       ref={overlayRef}
     >
-      <StyledDrawerSide
+      <aside
+        className={cx(
+          "box-border block",
+          "absolute bottom-0 top-0",
+          "h-full w-full",
+          "font-base",
+          "overflow-y-auto",
+          "shadow-raised",
+          "duration-slow transition-transform ease-in-out",
+          shown
+            ? "translate-z-0 translate-x-0 translate-y-0"
+            : "translate-z-full lm:translate-z-[var(--lm-drawer-width)] translate-x-0 translate-y-0",
```

Comment:
the animation of opening yet is not so smooth as current Drawer in master, I am working on make it work the same as there